README.md

```diff
@@ -222,70 +233,150 @@ buildctl build ... --output type=tar,dest=out.tar
 buildctl build ... --output type=tar > out.tar
 ```
 
-##### Exporting built image to Docker
+#### Docker tarball
 
 ```bash
 # exported tarball is also compatible with OCI spec
 buildctl build ... --output type=docker,name=myimage | docker load
 ```
 
-##### Exporting [OCI Image Format](https://github.com/opencontainers/image-spec) tarball to client
+#### OCI tarball
 
 ```bash
 buildctl build ... --output type=oci,dest=path/to/output.tar
 buildctl build ... --output type=oci > output.tar
 ```
+#### containerd image store
+
+The containerd worker needs to be used
+
+```bash
+buildctl build ... --output type=image,name=docker.io/username/image
+ctr --namespace=buildkit images ls
+```
+
+To change the containerd namespace, you need to change `worker.containerd.namespace` in [`/etc/buildkit/buildkitd.toml`](./docs/buildkitd.toml.md).
 
-### Exporting/Importing build cache (not image itself)
 
-#### To/From registry
+## Cache
 
+To show local build cache (`/var/lib/buildkit`):
+
+```bash
+buildctl du -v
+```
+
+To prune local build cache:
 ```bash
-buildctl build ... --export-cache type=registry,ref=localhost:5000/myrepo:buildcache
-buildctl build ... --import-cache type=registry,ref=localhost:5000/myrepo:buildcache
+buildctl prune
 ```
 
-#### To/From local filesystem
+### Garbage collection
+
+See [`./docs/buildkitd.toml.md`](./docs/buildkitd.toml.md).
+
+### Export cache
+
+BuildKit supports the following cache exporters:
+* `inline`: embed the cache into the image, and push them to the registry together
+* `registry`: push the image and the cache separately
+* `local`: export to a local directory
+
+In most case you want to use the `inline` cache exporter.
+However, note that the `inline` cache exporter only supports `min` cache mode. 
+To enable `max` cache mode, push the image and the cache separately by using `registry` cache exporter.
+
+#### Inline (push image and cache together)
+
+```bash
+buildctl build ... \
+  --export-cache type=inline \
+  --output type=image,name=docker.io/username/image,push=true
+```
+
+The inline cache is automatically imported when the cache is found in the destination image on the registry.
```

Comment:
This isn't true. You should still need `--import-cache type=registry`

Reply:
is this broken? https://github.com/moby/buildkit/blob/9c60d5be3e4eb59a8e8c4c7ab6b8884564c08516/hack/images#L81

Reply:
and wondering we can relax this limitation

Reply:
That part does not use previous cache, only exports. It's not run in ci except for releases(when we don't want to probably use cache).

Reply:
> and wondering we can relax this limitation

I think it's better to be explicit about it. There are use-cases where you only want to export and not import.

Reply:
updated